lib/install/management/rbac.go

```diff
@@ -16,11 +16,20 @@ package management
 
 import (
 	"context"
+	"fmt"
 
+	"net/url"
```

Comment:
`"net/url"` belongs in the standard pkg group above - therefore, there should be no newline between `"fmt"` and this import.

Reply:
Thanks. Done.